source/modules/MEI.cmn.xml

```diff
@@ -267,18 +267,15 @@
           <constraint>
             <sch:rule
               context="mei:beam[@place eq 'mixed' and not(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@staff != ./@staff] or descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@staff != ancestor::mei:staff/@n])]">
-              <sch:assert
-                test="count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@stem.dir]) = count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'])"
-                >Stem direction must be specified for all notes and chords under the
+              <sch:assert test="count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@stem.dir]) = count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'])" role="warning"
+                >Stem directions should be specified for all notes and chords under the
                 beam.</sch:assert>
-              <sch:assert
-                test="count(distinct-values(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord']/@stem.dir)) &gt; 1"
+              <sch:assert test="count(distinct-values(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord']/@stem.dir)) != 1"
```

Comment:
This would also be true for no stems, right?

Reply:
It is now that having no `@stem.dir` at all throws a warning, but not any longer an error. It is still an error having the same `@stem.dir` encoded everywhere.

Reply:
sounds as we decided in #690